lib/scrabble.rb

```diff
@@ -1,18 +1,47 @@
+require './lib/player'
 class Scrabble
+  attr_reader :players
 
-  def score(word)
-    1
+
+  def initialize(player = Player.new("Bob"))
+    @players = [player]
+    @point_values ={
```

Comment:
Would prefer you left this as a method since that was what was given. Should be able to use it basically the same way.